build/common.sh

```diff
@@ -501,9 +501,11 @@ function kube::build::ensure_data_container() {
   # If the data container exists AND exited successfully, we can use it.
   # Otherwise nuke it and start over.
   local ret=0
-  local code=$(docker inspect \
+  local code=0
+
+  code=$(docker inspect \
       -f '{{.State.ExitCode}}' \
-      "${KUBE_DATA_CONTAINER_NAME}" 2>/dev/null || ret=$?)
+      "${KUBE_DATA_CONTAINER_NAME}" 2>/dev/null) || ret=$?
```

Comment:
Will this code work in `set -o errexit` mode? Many script work in this mode, so keep it in mind.

Reply:
The script should not terminate, if that's what you mean. The potentially failing `docker inspect` command doesn't cause the script to exit because it's a non-final term in a `||` pipeline.